### ISSUE TITLE
Sober Graphics

### ISSUE THREAD

**Issue by vieir40** [vinegarhq/sober#425]
### Acknowledgement of checking existent issues

- [X] I have checked if my issue has already been reported, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

Its possible use the "Rtx" graphics in Sober? Whenever I try to use FFlagDebugForceFutureIsBrightPhase3 and DFIntDebugFRMQualityLevelOverride to have better graphics, roblox overrides the fflags

**Comment by CreateWithAce** [comment]
I think its because we are basing off a android system, you can try this with openGL, and other renderers to see if this fixes your issue.

**Comment by CreateWithAce** [comment]
also, this is not a bug. you should use the enhancement label.

**Comment by vieir40** [comment]
> I think its because we are basing off a android system, you can try this with openGL, and other renderers to see if this fixes your issue.
My problem is because Sober rewrites the ClientSettings to the default, is there any way to change the ClientSettings?

**Comment by CreateWithAce** [comment]
> > I think its because we are basing off a android system, you can try this with openGL, and other renderers to see if this fixes your issue.
> 
> My problem is because Sober rewrites the ClientSettings to the default, is there any way to change the ClientSettings?

Yes, its called the save button. you dont just exit it out, and if that is not the issue try and use a different text editor.

**Comment by vieir40** [comment]
> > > I think its because we are basing off a android system, you can try this with openGL, and other renderers to see if this fixes your issue.
> > 
> > 
> > My problem is because Sober rewrites the ClientSettings to the default, is there any way to change the ClientSettings?
> 
> Yes, its called the save button. you dont just exit it out, and if that is not the issue try and use a different text editor.

It rewrites no matter what i do, i tried to make the file immutable, its works, the file is not rewritten, but the flags dont work, if i remove it, it rewrites again. I was looking at other issues, and i found a problem similar to mine https://github.com/vinegarhq/sober/issues/20#issuecomment-2270184667

**Comment by tunis4** [comment]
on launch sober reformats the fflags file, adds default fflags and removes deprecated fflags. the ones that you are trying to add aren't getting removed

**Comment by CreateWithAce** [comment]
> > > > I think its because we are basing off a android system, you can try this with openGL, and other renderers to see if this fixes your issue.
> > > 
> > > 
> > > My problem is because Sober rewrites the ClientSettings to the default, is there any way to change the ClientSettings?
> > 
> > 
> > Yes, its called the save button. you dont just exit it out, and if that is not the issue try and use a different text editor.
> 
> It rewrites no matter what i do, i tried to make the file immutable, its works, the file is not rewritten, but the flags dont work, if i remove it, it rewrites again. I was looking at other issues, and i found a problem similar to mine [#20 (comment)](https://github.com/vinegarhq/sober/issues/20#issuecomment-2270184667)

Ok then i am sure you have the formatting wrong. The formating should look like this. (This is my clientappsettings.json)

{
    "DFFlagDebugPerfMode": "True",
    "DFFlagDisableDPIScale": true,
    "DFIntTaskSchedulerTargetFps": "5588562",
    "FFlagAdServiceEnabled": false,
    "FFlagCommitToGraphicsQualityFix": "True",
    "FFlagDebugDisableTelemetryEphemeralCounter": true,
    "FFlagDebugDisableTelemetryEphemeralStat": true,
    "FFlagDebugDisableTelemetryEventIngest": true,
    "FFlagDebugDisableTelemetryPoint": true,
    "FFlagDebugDisableTelemetryV2Counter": true,
    "FFlagDebugDisableTelemetryV2Event": true,
    "FFlagDebugDisableTelemetryV2Stat": true,
    "FFlagFutureIsBrightPhase3Vulkan": true,
    "FFlagGameBasicSettingsFramerateCap5": true,
    "FFlagRenderCheckThreading": "True",
    "FFlagTaskSchedulerLimitTargetFpsTo2402": "False",
    "FFlagTextureDeduplicationByHash4": false,
    "FIntDebugForceMSAASamples": 4,
    "FLogFMOD": 0
}
